pages/docs/tutorials/kafka/managing-schemas-using-schema-registry.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Managing schemas using Schema Registry
+description: A tutorial teaching how to manage schemas using Schema Registry.
+weight: 250
+---
+
+## Introduction
+In this tutorial, you'll learn how to manage schemas with Schema Registry. Managing schemas ensures a consistent data exchange across distributed systems, thus minimizing errors.
+
+## Background context
+The need for schema management has become increasingly prevalent to handle the evolving complexity of modern Event-Driven Architecture. A Schema Registry is a centralized service that stores and maintains schemas for data exchanged between various components of a modern distributed system. Validating exchanged data maintains data consistency and compatibility. 
+
+While several Schema Registry implementations exist, you will use the [Apicurio Registry](https://www.apicur.io/registry/) for this tutorial. Apicurio Registry is a popular open-source Schema Registry implementation that supports multiple serialization formats and facilitates schema management for diverse data in distributed systems. You will use Apicurio Registry combined with Avro, a language-neutral data serialization system.
+
+## Prerequisites
+[Install Docker](https://docs.docker.com/engine/install/) from the official website.
+
+
+### Start Apicurio Registry
+Start the Apicurio Registry locally with the following docker command:
+   
+```
+docker run --env CORS_ALLOWED_ORIGINS='*' -it -p 8080:8080 apicurio/apicurio-registry-mem:2.5.8.Final
+```
+
+### Upload Avro Schema
+Once your local instance of Apicurio Registry is running, upload your Avro schema. Open a new terminal window and create an Avro schema artifact with the following command:
+   
+```
+curl \
+http://localhost:8080/apis/registry/v2/groups/my-group/artifacts \
+-X POST  \
+-H "Content-Type: application/json; artifactType=AVRO" \
+-H "X-Registry-ArtifactId: UserSignedUp" \
+--data @- << EOF
+{
+  "type": "record",
+  "name": "UserSignedUp",
+  "namespace": "com.company",
+  "doc": "User sign-up information",
+  "fields": [
+    {
+      "name": "userId",
+      "type": "int"
+    },
+    {
+      "name": "userEmail",
+      "type": "string"
+    }
+  ]
+}
+EOF
+```
+
+<Remember>
+Download your Avro schema by visiting the following URL: 
+http://localhost:8080/apis/registry/v2/groups/my-group/artifacts/UserSignedUp.
+</Remember>
+
+### Replace link using JSON pointer
```

Comment:
it is not json pointer and it is not about replacing link

```suggestion
### Update schema reference
```

We miss a flow in this tutorial

Kafka tutorials were suppose to be "one after another flow" always next mentions, is related to previous one, like extending it

So in this tutorial we should mention Kafka Avro tutorial at the beginning and not here inside this section. Like it is done in other tutorials.

1. first say that "yo, in previous tutorial you added avro schema to your asyncapi document and now you will put this schema in central registry of schemas and reuse it from there"
2. next show the part that you modify, like in Kafka Avro tutorial, first show part from previous tutorial, how it looked before so you show which part you improve - and then you improve it


It is important we are consistent across tutorials

Reply:
@Arya-Gupta ⬆️

Reply:
@alequetzalli How does this sound as part of the  `Introduction`:
"In the previous tutorial, you've learned how to add avro schemas to your AsyncAPI document. Now, you will learn how to save your schema in a central Schema Registry and reuse it from there."

Reply:
@derberg I can have a heading called `AsyncAPI document with Avro Schema` right before the first step of the tutorial. 

The previous tutorial taught you how to write an AsyncAPI document for Kafka messages using the Avro Schema. Here's an example of what an AsyncAPI document fully equipped with Avro Schema looks like:

`
asyncapi: 3.0.0
info:
  title: User Signup API
  version: 1.0.0
  description: The API notifies you whenever a new user signs up in the application.
servers:
  kafkaServer:
    host: test.mykafkacluster.org:8092
    description: Kafka Server
    protocol: kafka
operations:
  onUserSignedUp:
    action: receive
    channel:
      $ref: '#/channels/userSignedUp'
channels:
  userSignedUp:
    description: This channel contains a message per each user who signs up in our application.
    address: user_signedup
    messages:
      userSignedUp:
        $ref: '#/components/messages/userSignedUp'
components:
  messages:
    userSignedUp:
      payload:
        schemaFormat: 'application/vnd.apache.avro;version=1.9.0'
        schema:
          type: record
          name: UserSignedUp
          namespace: com.company
          doc: User sign-up information
          fields:
            - name: userId
              type: int
            - name: userEmail
              type: string
`

Reply:
Why is this marked as resolved? you still need to add mention of the previous tutorial in the introduction.. please add that extra context :)

Reply:
yes, you need to mention previous tutorial in the introduction.. please do so before requesting another review :P